pimd/pim_cmd.c

```diff
@@ -701,14 +702,14 @@ static void igmp_show_interfaces_single(struct pim_instance *pim,
 				pim_ifp->igmp_query_max_response_time_dsec);
 
 			lmqt_msec = PIM_IGMP_LMQT_MSEC(
-				pim_ifp->igmp_query_max_response_time_dsec,
-				igmp->querier_robustness_variable);
+				pim_ifp->igmp_specific_query_max_response_time_dsec,
+				pim_ifp->igmp_last_member_query_count);
 
 			ohpi_msec =
 				PIM_IGMP_OHPI_DSEC(
-					igmp->querier_robustness_variable,
-					igmp->querier_query_interval,
-					pim_ifp->igmp_query_max_response_time_dsec)
+				  igmp->querier_robustness_variable,
```

Comment:
why the different whitespace changes?  Please put back

Reply:
just tried to adjust in 80 characters, put it back again 👍